index.js

```diff
@@ -0,0 +1,65 @@
+const books = [];
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const displayContainer = document.querySelector('.container');
+const storage = () => {
+  localStorage.setItem('books', JSON.stringify(books));
+};
+const addBtn = document.querySelector('.add');
+addBtn.addEventListener('click', (event) => {
+  event.preventDefault();
+  if (title.value !== '' && author.value !== '') {
+    const book = {
+      title,
+      author,
+    };
+    books.push(book);
+    const div = document.createElement('div');
+    books.forEach((book, index) => {
+      div.innerHTML = `
+            <p>${book.title.value}</p>
+            <p>${book.author.value}</p>                                          
+            <button type="button" id="${index}" class="remove">Remove</button>   
+            <hr>`;
+      const removebtn = div.querySelector('.remove');
+      removebtn.addEventListener('click', () => {
+        books.splice(index, 1);
+        div.remove();
+        storage();
```

Comment:
- [ ] Let's create a function that removes a book from the collection (hint: you can use the [array filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method. It makes your code more readable and improves the separation of tasks.